### PR TITLE
Announce DjangoCon US 2025

### PR DESCRIPTION
**Pull request by PeterGrand**
Have not previewed in browser because the docker-compose command errors for me

### CODE REVIEW

**Review by jefftriplett — APPROVED** [review]
LGTM

File: _posts/2024-12-31-djangoconus-2025-announced.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+author_url: /about
+author_avatar: defna
+categories: ""
+date: 2024-12-31 12:30:00
+feature_image: 2016-07-20
+layout: post
+read_time: ""
+show_avatar: true
+show_related_posts: false
+title:  "DjangoCon US 2024 has been announced!"
```
Comment (jefftriplett) [inline _posts/2024-12-31-djangoconus-2025-announced.md]:
```suggestion
title:  "DjangoCon US 2025 has been announced!"
```

Comment (jefftriplett) [reply _posts/2024-12-31-djangoconus-2025-announced.md]: Oops, just need to update the year.

**Review by jefftriplett — CHANGES_REQUESTED** [review]
One year typo.

**Review by tim-schilling — APPROVED** [review]

**Review by jefftriplett — APPROVED** [review]

**Pull request merged**